src/Adapter/SpecificPrice/CommandHandler/AddSpecificPriceHandler.php

```diff
@@ -37,7 +38,8 @@
 use SpecificPrice;
 
 /**
- * Handles AddSpecificPriceCommand using legacy object model
+ * @deprecated since 1.7.9.0
```

Comment:
The command itself is deprecated since 1.7.8.0 So I think it should be the same for the handler (I just forgot it)
However the delate command is indeed deprecated in 1.7.9.0